app/services/atmosphere/optimizer.rb

```diff
@@ -94,7 +104,9 @@ def scale(hint)
             appl_manager.unsatisfied("Not enough vms to scale down")
           end
         end
+        action.log("Scaling finished")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.